Domains/HairdresserDomain/DTO/ServiceDto.cs

```diff
@@ -0,0 +1,11 @@
+
+namespace HairdresserAPI.HairdresserDomain.HairdresserDtos;
+
+public record ServiceDto
```

Comment:
All properties in this DTO class should have keyword `required `to ensure that data sent to API is valid.  Example
`public required string Name { get; set; }`